src/Components/EditAdditionalRepositoriesTable/EditAdditionalRepositoriesTable.js

```diff
@@ -0,0 +1,404 @@
+          <ToolbarFilter
+            chips={searchValue !== '' ? [searchValue] : []}
+            deleteChip={() => setSearchValue('')}
+            deleteChipGroup={() => setSearchValue('')}
+            categoryName="Name"
+            showToolbarItem={activeAttributeMenu === 'Name' || 'Label'}
+            variant="search-filter"
+          >
+            {toolbarSearchInput}
+          </ToolbarFilter>
+        </ToolbarGroup>
+        <ToolbarItem variant="pagination" align={{ default: 'alignRight' }}>
+          {pagination()}
+        </ToolbarItem>
+      </ToolbarContent>
+    </Toolbar>
+  );
+
+  /* Empty State filter search return rationale*/
+
+  const emptyState = (
+    <EmptyState>
+      <EmptyStateIcon icon={SearchIcon} />
+      <Title size="lg" headingLevel="h4">
+        No results found
+      </Title>
+      <EmptyStateBody>
+        No results match the filter criteria. Clear all filters and try again.
+      </EmptyStateBody>
+      <EmptyStatePrimary>
+        <Button
+          variant="link"
+          onClick={() => {
+            setSearchValue('');
+          }}
+        >
+          Clear all filters
+        </Button>
+      </EmptyStatePrimary>
+    </EmptyState>
+  );
+
+  if (repositories && repositories.length > 0) {
+    return (
+      <React.Fragment>
+        {editAdditionalReposToolbar}
+        <TableComposable
+          aria-label="Additional Repositories Selectable Table"
+          variant="compact"
+        >
+          <Thead>
+            <Tr ouiaSafe={true}>
+              <Th />
+              <Th sort={getSortParams(0)} width={50}>
+                {columnNames.repositoryName}
+              </Th>
+              <Th sort={getSortParams(1)}>{columnNames.repositoryLabel}</Th>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {paginatedRepos.map((repositories, rowIndex) => (
+              <Tr key={(repositories, rowIndex)} ouiaSafe={true}>
+                <Td
+                  select={{
+                    rowIndex,
+                    onSelect: (_event, isSelecting) =>
+                      onSelectRepo(repositories, rowIndex, isSelecting),
+                    isSelected: isRepoSelected(repositories),
+                    disable: !isRepoSelectable(repositories),
+                  }}
+                />
+                <Td dataLabel={columnNames.repositoryName}>
+                  {repositories.repositoryName}
+                </Td>
+                <Td dataLabel={columnNames.repositoryLabel}>
+                  {repositories.repositoryLabel}
+                </Td>
+              </Tr>
+            ))}
+            {paginatedRepos.length == 0 && (
+              <Tr>
+                <Td colSpan={8}>
+                  <Bullseye>{emptyState}</Bullseye>
+                </Td>
+              </Tr>
+            )}
+          </Tbody>
+        </TableComposable>
+        {pagination(PaginationVariant.bottom)}
+      </React.Fragment>
+    );
```

Comment:
Since you're passing repositories as an prop -  I think it would be better to extract Loading/Unavailable logic to parent component. Or alternatively pass activation_key as prop and fetch repositories inside this component, that way you will have access to isLoading and error state from react-query